src/components/form/validators/validate-date.ts

```diff
@@ -1,6 +1,6 @@
 import isEmptyControlValue from './is-empty-control-value';
 
-export default function validateDate(value) {
+export default function validateDate(value: any) {
```

Comment:
```suggestion
export default function validateDate(value: any):string {
```
Maybe add a return type like all other validators?